docs/book/src/cronjob-tutorial/testdata/project/config/prometheus/monitor.yaml

```diff
@@ -11,8 +11,11 @@ metadata:
 spec:
   endpoints:
     - path: /metrics
-      port: http # Ensure this is the name of the port that exposes HTTP metrics
-      scheme: http
+      port: https # Ensure this is the name of the port that exposes HTTP metrics
```

Comment:
If I do not want to use the c+r implementation and instead I want to use security policies + cert-manager, I am be able to use it with this default scaffold?